src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Commands.Automation.Test
+{
+    using Microsoft.Azure.Commands.Automation.Test;
+    using Microsoft.Azure.ServiceManagement.Common.Models;
+    using Microsoft.WindowsAzure.Commands.ScenarioTest;
+    using Xunit;
+
+    public class UpdateDynamicGroupPrePostTests : AutomationScenarioTestsBase
```

Comment:
@jemex the build is failing partially because you are inheriting from this `AutomationScenarioTestBase` class that doesn't exist anymore -- you should be inheriting from `AutomationTestRunner`

Reply:
@cormacpayne , Thank you so much, I wasn't able o figure it out. now is inheriting correctly